### ISSUE TITLE
Could not pass event PlayerPortalEvent to Multiverse-NetherPortals

### ISSUE THREAD

**Issue by mibby** [Multiverse/Multiverse-NetherPortals#132]
Paper dev 1552 (Spigot 1.12.2)
Multiverse-Core dev 738
Multiverse-NetherPortals dev 729

@dumptruckman This error spammed until my server crashed due to an out of memory error.

```
[20:19:42] [Server thread/ERROR]: Could not pass event PlayerPortalEvent to Multiverse-NetherPortals v2.5.2-SNAPSHOT-b729
org.bukkit.event.EventException: null
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:72) ~[patched_1.12.2.jar:git-Paper-1552]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.12.2.jar:git-Paper-1552]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.12.2.jar:git-Paper-1552]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:527) ~[patched_1.12.2.jar:git-Paper-1552]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:512) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.PlayerList.changeDimension(PlayerList.java:799) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.EntityPlayer.b(EntityPlayer.java:654) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.BlockEnderPortal.a(BlockEnderPortal.java:46) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.Entity.checkBlockCollisions(Entity.java:1015) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.Entity.move(Entity.java:596) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.EntityLiving.a(EntityLiving.java:1856) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.EntityHuman.a(EntityHuman.java:1454) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.EntityLiving.n(EntityLiving.java:2166) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.EntityHuman.n(EntityHuman.java:356) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.EntityLiving.B_(EntityLiving.java:1994) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.EntityHuman.B_(EntityHuman.java:146) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.EntityPlayer.playerTick(EntityPlayer.java:339) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.PlayerConnection.e(PlayerConnection.java:144) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.aufdemrand.denizen.nms.impl.packets.handlers.AbstractListenerPlayIn_v1_12_R1.e(AbstractListenerPlayIn_v1_12_R1.java:76) ~[?:?]
	at net.minecraft.server.v1_12_R1.NetworkManager.a(NetworkManager.java:255) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.ServerConnection.c(ServerConnection.java:150) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:948) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:423) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:774) ~[patched_1.12.2.jar:git-Paper-1552]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:666) ~[patched_1.12.2.jar:git-Paper-1552]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
Caused by: java.lang.IllegalArgumentException: Cannot use null location!
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:122) ~[patched_1.12.2.jar:git-Paper-1552]
	at org.bukkit.event.player.PlayerMoveEvent.validateLocation(PlayerMoveEvent.java:91) ~[patched_1.12.2.jar:git-Paper-1552]
	at org.bukkit.event.player.PlayerMoveEvent.setTo(PlayerMoveEvent.java:86) ~[patched_1.12.2.jar:git-Paper-1552]
	at com.onarandombox.MultiverseNetherPortals.listeners.MVNPPlayerListener.onPlayerPortal(MVNPPlayerListener.java:90) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor1004.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:70) ~[patched_1.12.2.jar:git-Paper-1552]
	... 25 more
```

**Comment by kaniesha** [comment]
Having a similar error on git-Spigot-093165d-ba575a5  ver 1.14.2-R0.1-SNAPSHOT.
https://gist.github.com/kaniesha/2ea53b6caa7aed1204da21a4a9c3ea17

**Comment by Jadiction** [comment]
Hi, this was already fixed in 4.0.0. Please update Multiverse-Core, Multiverse-NetherPortals and any other Multiverse plugins you may have. Closing issue because it's an old post. If you are still running a Minecraft server and this persists, please don't hesitate to ask for support on our [Discord](https://discord.gg/NZtfKky) (quickest support) or submit a new issue.
